Source/WebCore/css/StyleProperties.cpp

```diff
@@ -1729,6 +1802,11 @@ StringBuilder StyleProperties::asTextInternal() const
             case CSSPropertyContainIntrinsicWidth:
                 shorthandPropertyID = CSSPropertyContainIntrinsicSize;
                 break;
+            case CSSPropertyGridTemplateRows:
+            case CSSPropertyGridTemplateColumns:
+            case CSSPropertyGridTemplateAreas:
+                shorthandPropertyID = CSSPropertyGridTemplate;
```

Comment:
It's fine to leave it for a follow-up, but this should try to serialize with the `grid` shorthand if possible. You can copy the `serializeBorderShorthand` hack to handle the multiple shorthands.